localstack/services/awslambda/invocation/runtime_environment.py

```diff
@@ -91,7 +95,6 @@ def get_environment_variables(self) -> Dict[str, str]:
             # https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html
             # a) Reserved environment variables
             # _HANDLER conditionally added below
-            # TODO: _X_AMZN_TRACE_ID
```

Comment:
nit: would be nice to keep it as documentation for completeness with https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html